pandas/core/internals/managers.py

```diff
@@ -998,6 +1001,14 @@ def iget(self, i: int) -> "SingleBlockManager":
             fastpath=True,
         )
 
+    def iget_values(self, i: int):
```

Comment:
`-> ArrayLike`

If we do implement this new method, let's use it in `iget`

Reply:
> If we do implement this new method, let's use it in iget

I did that first, but `iget` needs both block and values for the rest of its function, so using it there doesn't duplicate any code